### PR TITLE
LRQA-82932 Update test case priorities

### PR DESCRIPTION
**Pull request by gaborlovasdotcom**
[LRQA-82932](https://liferay.atlassian.net/browse/LRQA-82932)

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase [hunk 1]
```diff
@@ -806,7 +806,7 @@ definition {
 		Staging.viewNoPublishContentCountViaPortletStagingCP(stagingDateRange = "From Last Publish Date");
 	}
 
-	@priority = 4
+	@priority = 3
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase:809]: This is a kind of smoke test for publication, it's more reasonable to be FP4.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentStagingUpgrade.testcase [hunk 2]
```diff
@@ -15,7 +15,7 @@ definition {
 		SearchAdministration.executeReindex();
 	}
 
-	@priority = 5
+	@priority = 3
```
Comment (JoyceWang08211) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentStagingUpgrade.testcase:18]: I think upgrade tests should be FP4 at least.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 3]
```diff
@@ -1098,6 +1100,7 @@ definition {
 	}
 
 	@description = "This is a use case for LRQA-47381. User can authorize external applications."
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase]: Is FP5 acceptable for this?

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase [hunk 4]
```diff
@@ -1166,6 +1169,7 @@ definition {
 	}
 
 	@description = "This is a use case for LRQA-41000. User can view and revoke granted authorizations."
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase]: FP5 acceptable?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase [hunk 5]
```diff
@@ -264,7 +264,7 @@ definition {
 	}
 
 	@description = "Verify is possible edit an exported template."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase:267]: Set

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase [hunk 6]
```diff
@@ -377,7 +377,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-82752. Verify is possible export content and then publish."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase:380]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase [hunk 7]
```diff
@@ -1328,7 +1328,7 @@ definition {
 	}
 
 	@description = "Verify is possible export/import a Smoke template."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase:1331]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImportPortlet.testcase [hunk 8]
```diff
@@ -453,7 +453,7 @@ definition {
 	}
 
 	@description = "Verify is possible export import deletions."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImportPortlet.testcase:456]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImportPortlet.testcase [hunk 9]
```diff
@@ -509,7 +509,7 @@ definition {
 	}
 
 	@description = "Verify is possible export/import a Portlet."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImportPortlet.testcase:512]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 10]
```diff
@@ -21,7 +21,7 @@ definition {
 	}
 
 	@description = "This is a use case for LRQA-29297. Verify that Admin user can delete a custom export process."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:24]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 11]
```diff
@@ -37,7 +37,7 @@ definition {
 	}
 
 	@description = "This is a use case for LRQA-29297. Verify that the Admin user can delete a portlet export process."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:40]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase [hunk 12]
```diff
@@ -61,7 +61,7 @@ definition {
 	}
 
 	@description = "This is a use case for LRQA-47649. Verify that Admin user can relaunch a custom export process."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportProcess.testcase:64]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportTemplate.testcase [hunk 13]
```diff
@@ -34,7 +34,7 @@ definition {
 	}
 
 	@description = "Verify is possible delete an export template."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportTemplate.testcase:37]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/upgrades/ExportImportUpgrade.testcase [hunk 14]
```diff
@@ -15,7 +15,7 @@ definition {
 	}
 
 	@description = "This is a test for LPS-138634. Relaunch export process for default site."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/upgrades/ExportImportUpgrade.testcase:18]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 15]
```diff
@@ -27,7 +27,7 @@ definition {
 	}
 
 	@description = "User can add site templates to the portal."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:30]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 16]
```diff
@@ -225,7 +225,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-150460 TC-1. User is able to trigger propagation of the site template to all connected sites automatically."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:228]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 17]
```diff
@@ -110,7 +110,7 @@ definition {
 	}
 
 	@description = "User can withdraw site administrator's right to modificate of pages on a site template."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:113]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 18]
```diff
@@ -140,7 +140,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-121137. Admin can enable and disable the porpagation of site template changes on site templates admin page."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:143]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 19]
```diff
@@ -866,7 +866,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-87081. User can delete site template both from it's hamburger menu and from the list of the site template admin page."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:869]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 20]
```diff
@@ -999,7 +999,7 @@ definition {
 	}
 
 	@description = "User can edit an existing site template."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:1002]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase [hunk 21]
```diff
@@ -1031,7 +1031,7 @@ definition {
 	}
 
 	@description = "User can edit the status of an existing site template."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/cpsitetemplates/CPSitetemplates.testcase:1034]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/upgrades/SitetemplatesUpgrade.testcase [hunk 22]
```diff
@@ -15,7 +15,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-174465 and LPS-174471. TC-7 Verify that the exclamation mark are present after upgrade."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/sitetemplates/upgrades/SitetemplatesUpgrade.testcase:18]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStaging.testcase [hunk 23]
```diff
@@ -148,7 +148,7 @@ definition {
 			pageStaging = "true");
 	}
 
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStaging.testcase:151]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStaging.testcase [hunk 24]
```diff
@@ -402,7 +402,7 @@ definition {
 		AssertElementNotPresent(locator1 = "Staging#PROCESS_HALF_DISPLAY_PAGE");
 	}
 
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStaging.testcase:405]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStagingWithVersioning.testcase [hunk 25]
```diff
@@ -64,7 +64,7 @@ definition {
 	}
 
 	@description = "Verify is possible add a page variation."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStagingWithVersioning.testcase:67]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStagingWithVersioning.testcase [hunk 26]
```diff
@@ -145,7 +145,7 @@ definition {
 	}
 
 	@description = "This is a test for LPS-36620. Verified that navigation is visible when clicking the site page variation."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStagingWithVersioning.testcase:148]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStagingWithVersioning.testcase [hunk 27]
```diff
@@ -109,7 +109,7 @@ definition {
 	}
 
 	@description = "Verify is possible add page with versioning enabled."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/pgstaging/PGStagingWithVersioning.testcase:112]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecase.testcase [hunk 28]
```diff
@@ -1191,7 +1191,7 @@ definition {
 		}
 	}
 
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecase.testcase:1194]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/upgrades/StagingUpgrade.testcase [hunk 29]
```diff
@@ -37,7 +37,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-98651."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/upgrades/StagingUpgrade.testcase:40]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecase.testcase [hunk 30]
```diff
@@ -2967,7 +2967,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-76633."
-	@priority = 5
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecase.testcase:2970]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase [hunk 31]
```diff
@@ -139,7 +139,7 @@ definition {
 		PagesAdmin.viewPageItemPresent(pageName = "Test Page Edit");
 	}
 
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase:142]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase [hunk 32]
```diff
@@ -175,7 +175,7 @@ definition {
 	}
 
 	@description = "This is a use case for LPS-78292."
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase:178]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase [hunk 33]
```diff
@@ -473,7 +473,7 @@ definition {
 			entryTitle = "Tavaszi felhők");
 	}
 
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase:476]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase [hunk 34]
```diff
@@ -534,7 +534,7 @@ definition {
 		Portlet.viewSpecificPG(portletName = "Site Map");
 	}
 
-	@priority = 4
+	@priority = 3
```
[the remark this answers is not in the document]
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase:537]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase (hunk 1 above)
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/StagingUsecaseWithVersioning.testcase:809]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentStagingUpgrade.testcase (hunk 2 above)
Comment (gaborlovasdotcom) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/upgrades/webcontent/WebContentStagingUpgrade.testcase:18]: Set to FP4

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase (hunk 3 above)
[the remark this answers is not in the document]
Comment (JoyceWang08211) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase]: Yes.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase (hunk 4 above)
[the remark this answers is not in the document]
Comment (JoyceWang08211) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase]: Yes

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**